.github/workflows/release.yml

```diff
@@ -49,7 +66,8 @@ jobs:
         run: make test
   update-api-specs:
     runs-on: ubuntu-latest
-#     needs: [test]
+    needs: [ token-validation ]
```

Comment:
test already needs `token-validation` , and update-api-specs: needs `test`, why are we changing this to `token-validation ` here?

Reply:
+1

Reply:
We're currently disabling the dependency on test. Which is why I added it. Once we enable the dependency on test, we don't need the dependency on token-validation as well. I've changed that now